gnosis/cowswap
# [presig] Presign orders

# Summary

![image](https://user-images.githubusercontent.com/2352112/131551276-6ceba4d9-1366-4c20-8419-2726fc547a39.png)

This PR allows to send the pre-sign transaction.

It readjust 

For now, the UX is improbable but you can already send valid presigned transactions with this PR.

## not included
I would like that we try to consolidate the two transactions into one to provide a better UX. This would not be included in this PR.

# To Test

1. Sign an order
2. Observe that now you create 2 items in your recent transactions. One is the order, the other is the transaction of the presign.
